src/components/FontChangeButton.js

```diff
@@ -1,35 +1,60 @@
 
 
 class FontChangeButton extends React.Component {
-  constructor() {
-    super()
-    this.state={ active: true }
+  constructor(props) {
+    super(props)
+    this.state={ active: false, initialize:true }
     this.anima = this.anima.bind(this)
   }
   anima() {
     this.setState({
-      active: !this.state.active
+      active: !this.state.active,
+      initialize:false
     })
   }
   changeFontSize(fontSize) {
     this.props.changeFontSize(fontSize)
   }
+
   render() {
-    let animaStyle = styles['animaStyle-hide']
-    if(this.state.active ===true) {
-      animaStyle = styles['animaStyle-hide']
-    }else {
-      animaStyle = styles['animaStyle-show']
+    // font animation
+    let animaWrapper, animaButton, controlButtonIcon=fontIconClose
+    if(this.state.active===false && this.state.initialize===false) {
+      controlButtonIcon = fontIconClose
+      animaWrapper =   animaWrapper = styles['animaStyle-wrapper-hide']
+      animaButton = styles['anima-hide']
+    }else if(this.state.active===true && this.state.initialize===false) {
+      controlButtonIcon = fontIconOpen
+      animaWrapper = styles['animaStyle-wrapper-show']
+      animaButton = styles['anima-show']
     }
+
+    // font status
+    let fontSize = this.props.fontSize
     return(
       <span className={styles['button-appearance']}>
-       <button type="button" onClick={this.anima}>Fo</button>
-       <button type="button" onClick={()=>this.changeFontSize('small')} className={animaStyle}>A-</button>
-       <button type="button" onClick={()=>this.changeFontSize('medium')} className={animaStyle}>A</button>
-       <button type="button" onClick={()=>this.changeFontSize('large')} className={animaStyle}>A+</button>
-      </span>
+       <div className={classNames(styles['image-container'],styles['control-button'])} onClick={this.anima}>
+         <img src={controlButtonIcon}/>
+       </div>
+       <div className={classNames(animaWrapper,styles['wrapper'])}></div>
+       <div onClick={()=>this.changeFontSize('large')} className={classNames(styles['image-container'],animaButton,styles['image-container-large'])}>
+         <img className={classNames(styles['image-large'],fontSize === 'large' ? styles['font-status-active'] : styles['font-status-inactive'])} src={fontIconLarge}/>
+       </div>
+       <div onClick={()=>this.changeFontSize('medium')} className={classNames(styles['image-container'],animaButton,styles['image-container-medium'])}>
+         <img className={classNames(styles['image-medium'],fontSize === 'medium' ? styles['font-status-active'] : styles['font-status-inactive'])} src={fontIconMedium}/>
+       </div>
+       <div onClick={()=>this.changeFontSize('small')} className={classNames(styles['image-container'],animaButton,styles['image-container-small'])}>
+         <img  className={classNames(styles['image-small'],fontSize === 'small' ? styles['font-status-active'] : styles['font-status-inactive'])} src={fontIconSmall}/>
```

Comment:
could you give these img tags for buttons the ```styles['font-status-inactive']``` style as a part of default styling?
So you will not need write ```styles['font-status-inactive']```, just need  ```styles['font-status-active']```